docs/governance/how-is-etherlink-governed.md

```diff
@@ -21,12 +21,12 @@ The Etherlink kernel governance process is a streamlined version of the [Tezos g
 It consists of three periods: a Proposal period and a Promotion period, which are supervised by Etherlink's kernel governance contract, and a Cooldown period, which is enforced by the Etherlink kernel itself.
 
 The lengths of these periods are stored in the [kernel governance contract](https://better-call.dev/mainnet/KT1FPG4NApqTJjwvmhWvqA14m5PJxu9qgpBK).
-This table shows the period lengths as of the Bifröst Etherlink update and the Tezos Quebec protocol:
+This table shows the period lengths as of the Calypso Etherlink update and the Tezos Quebec protocol:
 
 Period | Length | Approximate time
 --- | --- | ---
-Proposal | 20480 layer 1 blocks | About 2 days
-Promotion | 20480 layer 1 blocks | About 2 days
+Proposal | 30720 layer 1 blocks | About 3 days
```

Comment:
Alternatively, you could say “At least 2 days, 20 hours and 16 minutes” maybe?
No strong opinion, I think your proposal has the merit of clarity.

Reply:
I'll address block drift in a future PR.